src/documents/IoCTestingFramework.md

```diff
@@ -122,151 +116,422 @@ fluid.tests.globalCatTest = function (catt) {
-    }
-];
+In the very [simple example](#simple-example) above, the member `sequence` is omitted since there is just one fixture record. 
+This record executes a global function, `fluid.tests.globalCatTest` which makes one jqUnit assertion.
+In a realistic use case, each entry within `tests` will instead hold an entry named `sequence` which holds an array of fixture records
+representing sequence points to be attained by the test case - as seen in the [asyncTester example](#-testcaseholder-demonstrating-sequence-record) below. 
 
-// A helper function which just returns the global -
-// realistically, it would assemble a sequence using more complex logic
-fluid.tests.getCatModules = function () {
-    return fluid.tests.catTesterModules;
-};
+As well as containing a flat list of fixture records, `sequence` may also contain nested arrays of such records. These nested arrays will be
+flattened into a single array by use of the utility [`fluid.flatten`](CoreAPI.md#fluid-flatten-array-) before being processed. This helps in
+assembling complex sequences out of previously canned sequence segments. However, building up complex, reusable test sequences is best done by
+use of the [`sequenceGrade`](/IoCTestingFramework.md#using-sequencegrade-to-build-up-complex-reusable-test-sequences) element, instead of the `sequence` element.
 
-fluid.defaults("fluid.tests.catTester", {
-    gradeNames: ["fluid.test.testCaseHolder"],
-    moduleSource: {
-        funcName: "fluid.tests.getCatModules"
-    }
-});
-```
+An example of the `sequence` element appears below in the [asyncTester example](#-testcaseholder-demonstrating-sequence-record) below.
 
+In order to run this test case, we can either simply construct an instance of the environment tree by calling `fluid.tests.myTestTree()`,
+or submit its name to the global driver function `fluid.test.runTests` as `fluid.test.runTests("fluid.tests.myTestTree")`.
+The latter method should be used when running multiple environments within the same file to ensure that their execution is properly serialised.
 
 ### Supported fixture records ###
 
-The IoC testing system currently supports the following 5 types of fixture record, which can be assigned to two categories -
+The IoC testing system currently supports the following 6 types of fixture record, which can be assigned to two categories -
 "executors", which actively trigger an action, and "binders" which register some form of listener in order to receive an event from
 the tree under test. These are recognised using a "duck typing system" similar to that used in the Fluid Renderer. These records may
 either form the complete payload for a test held in the `tests` section of a `TestCaseHolder`, or may appear as elements of an array held
 in its `sequence` member, representing a sequence of actions (either executors or binders) to be performed by the test case.
 
+
+<style>
+<!-- This block is stripped out by the github markdown processor, but retained by docpad -->
```

Comment:
This is actually appearing in GitHub's rendering of the file. https://github.com/amb26/infusion-docs/blob/e3c9f87ecb064789a5d214b5b40a17e4f2129a2d/src/documents/IoCTestingFramework.md#supported-fixture-records